packages/amplify-provider-awscloudformation/src/admin-modelgen.ts

```diff
@@ -26,6 +27,13 @@ export const adminModelgen = async (context: $TSContext, resources: $TSAny[]): P
     return;
   }
 
+  const localSchemaPath = path.join(pathManager.getResourceDirectoryPath(undefined, 'api', resourceName), 'schema.graphql');
+  // Early return with useful warning if the schema file does not exist
+  if (!fs.existsSync(localSchemaPath)) {
+    printer.warn(`Could not find the GraphQL schema file at "${localSchemaPath}". Amplify Studio's schema editor might not work as intended if you're using multiple schema files.`);
+    return;
+  }
```

Comment:
keep in mind that we generate these files regardless of customer using studio or not.
So for users who never opened Studio this warning might be confusing.